dask_cloudprovider/aws/ecs.py

```diff
@@ -953,6 +967,18 @@ async def _start(
                 or await self._create_security_groups()
             )
 
+        self._worker_task_def_name = (
+            f"{str(self._task_def_name)}-worker-{uuid}"
```

Comment:
I think you might've misunderstood my comment. This should just be the task def name, we should allow the user to construct whatever format string they like. If they place `{uuid}` in the string it will be replaced with the uuid.

```suggestion
            self._task_def_name
```